pkg/migration/controllers/migration.go

```diff
@@ -909,7 +909,7 @@ func (m *MigrationController) runPreExecRule(migration *stork_api.Migration, mig
 	}
 	terminationChannels := make([]chan bool, 0)
 	for _, ns := range migrationNamespaces {
-		r, err := storkops.Instance().GetRule(migration.Spec.PreExecRule, ns)
+		r, err := storkops.Instance().GetRule(migration.Spec.PreExecRule, migration.Namespace)
```

Comment:
The rule will remain same in all iterations of the for loop. This call can be moved outside the for loop to avoid calling it multiple times in case of migration schedule in admin namespace.

Reply:
Makes sense. Will address